hw03_karimh.py

```diff
@@ -8,20 +8,111 @@
 
 import turtle
 
-def function_1():
-    "include docstring"
+#Assigning name to the turtle
+Heer = turtle.Turtle()
 
+def tyre_1(rad, th):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
+    "Makes the left tyre of the car"
 
-def function_2():
-    "docstring"
+    #Gives black color to the tyre
+    Heer.fillcolor("black")
+    Heer.begin_fill()
 
-def function_3():
-    "docstring"
+    #Attributes of the tyre
+    Heer.pensize(th)
+    Heer.circle(rad)
+    Heer.end_fill()
+def tyre_2(rad, th):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
 
-def fucntion_4():
-    "docstring"
+    Heer.fillcolor("black")
+    Heer.begin_fill()
+
+    Heer.pensize(th)
+    Heer.circle(rad)
+    Heer.end_fill()
+def lower_body(l, w):
+    "l = length of the lower rectangular body of car"
+    "w = width of the lower rectangular body of car"
+    "Makes the lower body of the car which looks like a big rectangle"
+
+    #Colors the lower body blue
+    Heer.fillcolor("blue")
+    Heer.begin_fill()
+
+    #Gives movement directions to the turtle
+    Heer.left(180)
+    Heer.forward(l)
+    Heer.right(90)
+    Heer.forward(w)
+    Heer.right(90)
+    Heer.forward(l)
+    Heer.right(90)
+    Heer.forward(w)
+    Heer.end_fill()
+
+def upper_body(ul,uw):
+    "ul = length of the upper rectangular body"
+    "uw = width of the upper rectangular body"
+
+    Heer.fillcolor("grey")
+    Heer.begin_fill()
+
+    Heer.left(180)
+    Heer.forward(ul)
+    Heer.right(90)
+    Heer.forward(uw)
+    Heer.right(90)
+    Heer.forward(ul)
+    Heer.right(90)
+    Heer.forward(uw)
+    Heer.end_fill()
 
 def main ():
+    #Creates method for the screen
+    wn = turtle.Screen()
+    wn.bgpic("chitral-valley.gif")     #Adds a background picture
+
+    wn.colormode(255)
+    wn.bgcolor("lightblue")
+    Heer.speed(0)
+
+    #Moves the turtle head from the center to the porper starting place
+    Heer.penup()
+    Heer.goto(-150, -200)
+    Heer.pendown()
+
     #call functions
+    tyre_1(50, 5)   #calls function 1 (making tyre 1)
```

Comment:
After `Heer` is moved to `main()`,  you probably need to pass it to this function call (and catch it in the function definition above).

Reply:
I added a parameter (H) for Heer in all my functions and called it in main.